bin/ci/az-publish.sh

```diff
@@ -41,9 +43,13 @@ VERSION=$(xcrun strings calabash-dylibs/libCalabashFAT.dylib | grep -E "CALABASH
 # Evaluate the Xcode version used to build artifacts
 XC_VERSION=$(xcode_version)
 
-az --version
+# Evaluate calabash.framework SHASUM256
+FRAMEWORK_SHASUM256=$(shasum --algorithm 256 ${WORKING_DIR}/calabash.framework/calabash | cut -d " " -f 1)
 
-WORKING_DIR="${BUILD_SOURCESDIRECTORY}"
+# Evaluate dylibFAT SHASUM256
+DYLIBFAT_SHASUM256=$(shasum --algorithm 256 ${WORKING_DIR}/calabash-dylibs/libCalabashFAT.dylib | cut -d " " -f 1)
+
+az --version
```

Comment:
Do we really need it? :)

Reply:
It's interesting for me as well :)
This command has been here before, and no one knows why. Should I remove it?

Reply:
Ha!  It is left over from when we were exploring publishing to blob storage using the command line tools.

We can remove it or leave it.